src/Database/ValueBinder.php

```diff
@@ -67,8 +67,8 @@ public function bind($param, $value, $type = 'string')
     public function placeholder($token)
     {
         $number = $this->_bindingsCount++;
-        if ($token[0] !== ':' || $token !== '?') {
-            $token = sprintf(':c%s', $number);
+        if ($token[0] !== ':' && $token !== '?') {
+            $token = sprintf(':%s%s', $token, $number);
         }
```

Comment:
The if statement previously always returned true, and token was not being used in the generation.